stress/stress_submit.ml

```diff
@@ -0,0 +1,103 @@
+open Lwt.Infix
+open Lwt.Syntax
+
+let solve_to_custom req builder =
+  let params =
+    Yojson.Safe.to_string
+    @@ Solver_service_api.Worker.Solve_request.to_yojson req
+  in
+  let builder =
+    Solver_service_api.Raw.Builder.Solver.Solve.Params.init_pointer builder
+  in
+  Solver_service_api.Raw.Builder.Solver.Solve.Params.request_set builder params
+
+let solve cluster job request =
+  let action =
+    Cluster_api.Submission.custom_build
+    @@ Cluster_api.Custom.v ~kind:"solve"
+    @@ solve_to_custom request
+  in
+  let build_pool =
+    Current_ocluster.Connection.pool ~job ~pool:"solver" ~action ~cache_hint:""
+      cluster
+  in
+  Current.Job.start_with ~pool:build_pool job ~level:Current.Level.Average
+  >>= fun build_job ->
+  Capnp_rpc_lwt.Capability.with_ref build_job
+    (Current_ocluster.Connection.run_job ~job)
+
+let ocaml_package_name = "ocaml-base-compiler"
+let ocaml_version = "4.14.0"
+let opam_repository_commit = "34576e67c88137d40ce6ff9e252d549e9e87205f"
+
+let make_requests limit =
+  let* vars = Utils.get_vars ~ocaml_package_name ~ocaml_version () in
+  let* opam_packages = Utils.get_opam_packages () in
+  let rec requests packages acc n =
+    match packages with
+    | [] -> Lwt.return acc
+    | package :: tl ->
```

Comment:
Shouldn't this rather be a fold over the packages list? Maybe with `Lwt_list.fold_left_s`?
If not, you should reorder the arguments to 
```ocaml
  let rec requests acc n = function
    | [] -> Lwt.return acc
    | package :: tl ->
```

Reply:
Cool thanks.